posthog/queries/funnels/funnel_event_query.py

```diff
@@ -49,7 +44,7 @@ def get_query(
 
```

Comment:
can't comment on line 38: shouldn't we be using `self._person_id_alias` there too - we seem to be using person_id from events table, which is only PoEv1, but not PoEv2 compatible, no?

Reply:
yup, good spot!